keychain/tests/test_keychain.py

```diff
@@ -21,6 +24,7 @@ def setUp(self):
 
         self.keychain = self.env['keychain.account']
         config['keychain_key'] = Fernet.generate_key()
+        config['running_env'] = ""
```

Comment:
@andreparames IMO this commit must be removed... Indeed, you reset the running_env before before to save its previous value at the next line.

Reply:
You're right, I took the "Mig V10" commit and removed the actual migration parts, without realizing what was left was no longer relevant. Will remove the commit.